app/notification_push.go

```diff
@@ -236,11 +236,11 @@ func (a *App) clearPushNotificationSync(currentSessionId, userID, channelID, roo
 	msg.Badge = int(unreadCount)
 
 	if msg.IsCRTEnabled {
-		data, err := a.Srv().Store.Thread().GetThreadsForUser(userID, "", model.GetUserThreadsOpts{TotalsOnly: true})
+		totalUnreadMentions, err := a.Srv().Store.Thread().GetTotalUnreadMentions(userID, "", model.GetUserThreadsOpts{})
```

Comment:
This is the performance boost here -- we no longer count thread memberships nor unread threads only to discard this metadata.